hs_file_types/models/model_program.py

```diff
@@ -164,7 +164,7 @@ def set_field(term, field_name, obj, is_date=False):
             val = graph.value(subject=subject, predicate=term)
             if val:
                 if is_date:
-                    date = parser.parse(str(val))
+                    date = parser.parse(str(val)).date()
```

Comment:
I found an issue with model program release date parsing.  It was resulting in a datetime, now it is only a date.